metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableBackend.java

```diff
@@ -472,8 +472,12 @@ public Statistics getStatistics() {
             fetches.add(readRows.directTransform(result -> {
                 for (final FlatRow row : result) {
                     watcher.readData(row.getCells().size());
-                    List<Metric> metrics = Lists.transform(row.getCells(), transform);
+                    final List<Metric> metrics = Lists.transform(row.getCells(), transform);
                     metricsConsumer.accept(MetricCollection.build(type, metrics));
+
+                    if (metrics.size() > 1) {
+                        watcher.reportRowDensity(getDensity(metrics));
```

Comment:
Yeah, `1` as well. The desired goal would be to cause values to 'shift down' appropriately, so replacing them with something like `LONG.MAX_VALUE` could be a way.